decidim-admin/app/assets/javascripts/decidim/admin/application.js.es6

```diff
@@ -16,4 +16,3 @@ const pageLoad = () => {
 };
 
 $(document).on('turbolinks:load', pageLoad);
-$(pageLoad);
```

Comment:
`turbolinks:load` doesn't get called on first page load AFAIK, so this will break.